### PR TITLE
cleanup: resolve godot linter

### PR DESCRIPTION
**Pull request by yati1998**
This commit resolves godot linter issue
which says "Comment should end in a period (godot)".

Updates: #1586

Signed-off-by: Yati Padia <ypadia@redhat.com>

### CODE REVIEW

File: e2e/rbd.go [hunk 1]
```diff
@@ -56,7 +56,7 @@ var (
 )
 
 func deployRBDPlugin() {
-	// delete objects deployed by rook
+	// delete objects deployed by rook.
```
Comment (nixpanic) [inline e2e/rbd.go]: I think the comments-should-end-with-a-dot is only required for top-level comments, not inside functions? Could you check that please?

Comment (yati1998) [reply e2e/rbd.go]: Yes sure @nixpanic .  I will check and update accordingly.

**Comment by yati1998** [comment]
@nixpanic godot linter checks only for top-level comments. I have made the necessary changes, but we don't follow a particular format for comments throughout the code. Hence, I suggest we should have some guidelines that can bring uniformity to the code.

File: internal/rbd/driver.go [hunk 2]
```diff
@@ -124,7 +124,7 @@ func (r *Driver) Run(conf *util.Config) {
 	volJournal = journal.NewCSIVolumeJournal(CSIInstanceID)
 	snapJournal = journal.NewCSISnapshotJournal(CSIInstanceID)
 
-	// Initialize default library driver
+	// Initialize default library drivers
```
Comment (Yuggupta27) [inline internal/rbd/driver.go]: Did `godot` give a warning here? If not, I think this change can be reverted.

**Review by Yuggupta27 — CHANGES_REQUESTED** [review]

File: internal/rbd/driver.go (hunk 2 above)
Comment (yati1998) [reply internal/rbd/driver.go]: Hey, that's my mistake. Maybe added that while saving the file. will correct that. Thanks for notifying me.

**Review by Yuggupta27 — APPROVED** [review]

**Review by Madhu-1 — DISMISSED** [review]

**Comment by nixpanic** [comment]
> @nixpanic godot linter checks only for top-level comments. I have made the necessary changes, but we don't follow a particular format for comments throughout the code. Hence, I suggest we should have some guidelines that can bring uniformity to the code.

I think following the godot guidelines is sufficient. Comments inside functions are much more important to me than the formatting of them. The godot guidelines make sense for APIs that are re-used, and make documentation like https://pkg.go.dev/github.com/ceph/ceph-csi helpful for new contributors.

**Comment by nixpanic** [comment]
> /retest ci/centos/mini-e2e-helm/k8s-1.21

Failed to deploy minikube VM (#2264), [logs](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.21/detail/mini-e2e-helm_k8s-1.21/167/pipeline).

**Review by nixpanic — DISMISSED** [review]

**Review by nixpanic — DISMISSED** [review]

**Comment by nixpanic** [comment]
oh, sorry, #2270 introduced a conflict with this PR :disappointed:

**Comment by yati1998** [comment]
> oh, sorry, #2270 introduced a conflict with this PR

No worries will resolve it. Facing network issues currently.

**Review by Madhu-1 — APPROVED** [review]

**Pull request merged**